torch_xla/csrc/ops/ops.h

```diff
@@ -55,26 +55,14 @@ inline torch::lazy::NodePtr GenericOp(torch::lazy::OpKind op, xla::Shape shape,
                                         hash_seed);
 }
 
-torch::lazy::NodePtr Acos(const XlaValue& input);
-
-torch::lazy::NodePtr Acosh(const XlaValue& input);
-
```

Comment:
These were supposed to be deleted when we full codegen'ed `acos` and `acosh` in the last PR.